src/main/kotlin/org/rust/lang/core/stubs/StubImplementations.kt

```diff
@@ -269,6 +269,7 @@ fun factory(name: String): RsStubElementType<*, *> = when (name) {
     "LIFETIME" -> RsLifetimeStub.Type
     "LIFETIME_PARAMETER" -> RsLifetimeParameterStub.Type
     "FOR_LIFETIMES" -> RsPlaceholderStub.Type("FOR_LIFETIMES", ::RsForLifetimesImpl)
+    "TILDE_CONST" -> RsPlaceholderStub.Type("TILDE_CONST", ::RsTildeConstImpl)
```

Comment:
It looks like a stub node for `TILDE_CONST` is not needed because you just save `hasConst` to `RsPolyboundStub`